user-docs/latest/users-roles-permissions/configuring-administrator-roles.md

```diff
@@ -0,0 +1,160 @@
+:::
+
+## Editing a role
+
+![Administrator roles edition interface](../assets/users-permissions/administrator_roles-edition.png)
+
+The role edition interface allows to edit the details of an administrator role as well as configure in detail the permissions to all sections of your Strapi application. It is accessible from *Administration panel > Roles* either after clicking on the edit button ![icon edit](../assets/users-permissions/icon_edit.png) on the right side of a role's record, or after clicking on the **Add new role** button (see [Creating a new role](#creating-a-new-role)).
+
+::: warning ATTENTION
+It isn't possible to edit the permissions of the Super Admin role. All configurations are in read-only mode.
+:::
+
+### Editing role's details
+
+The details area of an administrator role editing interface allow to define the name of the role, and to give it a description that should help other administrators understand what the role gives access to.
+
+::: tip 💡 TIP
+In the top right corner, you can see a counter indicating how many administrators have been attributed the role.
+:::
+
+To edit a role's details, follow the instructions from the table below:
+
+| Role details  | Instructions                                                                                                                     |
+| ------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| Name          | Write the new name of the role in the textbox.                                                                                   |
+| Description   | Write the description of the role in the textbox.                                                                                |
+
+### Configuring role's permissions
+
+The permissions area of an administrator role editing interface allows to configure in detail what actions an administrator can do for any part of the Strapi application. It is displayed as a table, split into 4 categories: Collection Types, Single Types, Plugins and Settings.
+
+#### Collection and Single Types
+
+The Collection Types and Single Types categories respectively list all available collection and single types for the Strapi application. For each content type, the administrators can have the permission to perform the following actions: create, read, update, delete and publish.
+
+To configure Collection or Single Types permissions for a role:
+
+1. Go to the Collection Types or Single Types category of the permissions table.
+2. Tick the box on the left of the name of the content type to give access to. By default, all actions can be performed for all fields of the content type.
+3. (optional) Untick the action-related boxes to prevent actions of your choice.
```

Comment:
This one is EE-related, should we mention it?

Reply:
Yes, see: https://github.com/strapi/documentation/pull/118#discussion_r585079411